src/frontend/komponenter/Fagsak/Vilkårsvurdering/Registeropplysninger/RegisteropplysningerTabell.tsx

```diff
@@ -85,53 +112,57 @@ const RegisteropplysningerTabell: React.FC<IRegisteropplysningerTabellProps> = (
-                                                        Registeropplysning.SIVILSTAND ||
-                                                    opplysningstype ===
-                                                        Registeropplysning.DØDSBOADRESSE
-                                                        ? tilVisning(
-                                                              periode.fom
-                                                                  ? kalenderDato(periode.fom)
-                                                                  : undefined
-                                                          )
-                                                        : periodeToString({
-                                                              fom: periode.fom,
-                                                              tom: periode.tom,
-                                                          })
-                                                }
-                                            />
-                                        </TabellRad>
-                                    );
-                                })
-                        ) : (
+                        {manglerOpplysninger && (
                             <TabellRad key={`${opplysningstype}_ukjent`}>
                                 <td children={'Ingen opplysninger'} />
                                 <td children={'-'} />
                             </TabellRad>
                         )}
+                        {!manglerOpplysninger &&
+                            synligHistorikk.map(periode => (
+                                <TabellRad key={`${periode.fom}_${periode.tom}_${periode.verdi}`}>
+                                    <td children={periode.verdi} />
+                                    <td
+                                        children={
+                                            opplysningstype === Registeropplysning.SIVILSTAND ||
+                                            opplysningstype === Registeropplysning.DØDSBOADRESSE
+                                                ? tilVisning(
+                                                      periode.fom
+                                                          ? kalenderDato(periode.fom)
+                                                          : undefined
+                                                  )
+                                                : periodeToString({
+                                                      fom: periode.fom,
+                                                      tom: periode.tom,
+                                                  })
+                                        }
+                                    />
+                                </TabellRad>
+                            ))}
                     </tbody>
                 </Tabell>
             </Container>
+            {skalVæreEkspanderbar && (
```

Comment:
Burde vi ha lagt på !manglerOpplysninger her også? Skjønner at skalVæreEkspanderbar og manglerOpplysninger egentlig aldri vil være true på samme tid her, men hvis noen endrer uttrykket for en av de så kan det hende at det ikke stemmer lenger 🤔 hmm

Reply:
Både enig og uenig! Skal tenke litt på det en time

Reply:
La det til i evalueringen av `skalVæreEkspanderbar` på linje 91-94